### PR TITLE
Add project fonts

### PR DESCRIPTION
**Pull request by DavidBronfen**
#11

### CODE REVIEW

**Comment by DavidBronfen** [comment]
Can't implement fonts, Error thrown:
image

`@font-face` is not loaded correctly

File: client/src/assets/fonts.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@import '../styles';
+
+
+@font-face {
```
Comment (nirgn975) [inline client/src/assets/fonts.scss]: Why did you add `Arimo` font if you don't use it anywhere?
Are you plan to use it in the future?

**Comment by nirgn975** [comment]
I use https://github.com/hasadna/open_pension/blob/master/client/src/styles.scss

File: client/src/assets/fonts.scss (hunk 1 above)
Comment (DavidBronfen) [inline client/src/assets/fonts.scss]: It's gonna be the main font, I just check if it's loaded...

**Comment by DavidBronfen** [comment]
@nirgn975 So it's batter to use import (CDN) for the fonts I'll use? 
I thought the best way is to download the fonts and load them locally

**Comment by nirgn975** [comment]
@DavidBronfen 
If I remember correctly, if you use a CDN and the computer already download the font (because another side request it), the browser will not download it again and it'll take it from the local version. If you download and load them locally it'll happen every time.

 But check me :)

**Comment by nirgn975** [comment]
build__59_-_davidbronfen_retrogusto_-_travis_ci

As you can see, the test check that the text is `equal` and not `contains in..`. You can change this string, but I think it better to remove this test (you will create your tests in future commits, so I think it's okay).

**Pull request merged**